lib/oxidized/model/acos.rb

```diff
@@ -22,11 +29,12 @@ class ACOS < Oxidized::Model
     comment cfg
   end
 
-  cmd 'show running-config all-partitions' do |cfg|
+  cmd 'show running-config' do |cfg|
      cfg.gsub! /(Current configuration).*/, '\\1 <removed>'
      cfg.gsub! /(Configuration last updated at).*/, '\\1 <removed>'
      cfg.gsub! /(Configuration last saved at).*/, '\\1 <removed>'
      cfg.gsub! /(Configuration last synchronized at).*/, '\\1 <removed>'
+     cfg
```

Comment:
you don't need this `cfg` here, since `#gsub!` already return the mutated string, but not harm in keep it 😉